pyaedt/modules/Boundary.py

```diff
@@ -536,3 +537,448 @@ def update_assignment(self):
+    def use_custom_radiation_surface(self, value):
+        self.props["UseCustomRadiationSurface"] = value
+        self.update()
+
+    @property
+    def custom_radiation_surface(self):
+        """Set/Get the Far Field Radiation Surface FaceList."""
+        return self.props["CustomRadiationSurface"]
+
+    @custom_radiation_surface.setter
+    def custom_radiation_surface(self, value):
+        if value:
+            self.props["UseCustomRadiationSurface"] = True
+            self.props["CustomRadiationSurface"] = value
+        else:
+            self.props["UseCustomRadiationSurface"] = False
+            self.props["CustomRadiationSurface"] = ""
+        self.update()
+
+    @property
+    def use_local_coordinate_system(self):
+        """Set/Get the usage of a custom Coordinate System."""
+        return self.props["UseLocalCS"]
+
+    @use_local_coordinate_system.setter
+    def use_local_coordinate_system(self, value):
+        self.props["UseLocalCS"] = value
+        self.update()
+
+    @property
+    def local_coordinate_system(self):
+        """Set/Get the custom Coordinate System name."""
+        return self.props["CoordSystem"]
+
+    @local_coordinate_system.setter
+    def local_coordinate_system(self, value):
+        if value:
+            self.props["UseLocalCS"] = True
+            self.props["CoordSystem"] = value
+        else:
+            self.props["UseLocalCS"] = False
+            self.props["CoordSystem"] = ""
+        self.update()
+
+    @property
+    def polarization(self):
+        """Set/Get the Far Field Polarization."""
+        return self.props["Polarization"]
+
+    @polarization.setter
+    def polarization(self, value):
+        self.props["Polarization"] = value
+        self.update()
+
+    @property
+    def slant_angle(self):
+        """Set/Get the Far Field Slant Angle if Polarization is Set to `Slant`."""
+
+        if self.props["Polarization"] == "Slant":
+            return self.props["SlantAngle"]
+        else:
+            return
+
+    @slant_angle.setter
+    def slant_angle(self, value):
+        self.props["Polarization"] = "Slant"
+        self.props["SlantAngle"] = value
+        self.update()
+
+    @property
+    def theta_start(self):
+        """Set/Get the Far Field Theta Start Angle if Definition is Set to `Theta-Phi`."""
+
+        if "ThetaStart" in self.props:
+            return self.props["ThetaStart"]
+        else:
+            return
+
+    @property
+    def theta_stop(self):
+        """Set/Get the Far Field Theta Stop Angle if Definition is Set to `Theta-Phi`."""
+
+        if "ThetaStop" in self.props:
+            return self.props["ThetaStop"]
+        else:
+            return
+
+    @property
+    def theta_step(self):
+        """Set/Get the Far Field Theta Step Angle if Definition is Set to `Theta-Phi`."""
+
+        if "ThetaStep" in self.props:
+            return self.props["ThetaStep"]
+        else:
+            return
```

Comment:
@maxcapodi78 we could raise an exception for all those `return `instead of returning `None`.

Reply:
It is expected behavior of AEDT.